console/src/main/scripts/plugins/directives/vendor/moment/moment.d.ts

```diff
@@ -1,472 +0,0 @@
-// Type definitions for Moment.js 2.8.0
```

Comment:
We still need to add the rest of the d.ts files for all the other libraries in the project. Moment.d.ts is the only type def file.